app/models/account.rb

```diff
@@ -67,7 +67,7 @@ def self.xml_to_hashes(xmlNode, findPath, typeName)
       nh[:acctid] = nh.delete("#{typeName}id".to_sym)
       nh[:acctid] = nil unless nh[:acctid].respond_to?(:to_int) || nh[:acctid].to_s =~ /^-?[0-9]+$/
       # Convert to signed integer values for acctid
-      nh[:acctid] = [nh[:acctid].to_i].pack("I").unpack("i")[0] unless nh[:acctid].nil?
+      nh[:acctid] = [nh[:acctid].to_i].pack("I").unpack1("i") unless nh[:acctid].nil?
```

Comment:
oops

Reply:
I think that's actually correct - I agree it does look strange though.